packages/Input/src/Input.styles.js

```diff
@@ -30,7 +30,15 @@ const inputStyles = css`
 
     background-color: ${tokens.color.white};
     border: 1px solid ${tokens.border.color};
-    border-radius: ${tokens.border.radius};
+    border-radius: ${({ hasBorderRadius }) => {
```

Comment:
I think I would rather have the `<DateRangePicker>` override the styling of the `<Input>` instead of introducing this additional complexity here.  Similar to the way the `<ButtonGroup>` overrides the styling of the `<Buttons>` it is composed of:

https://github.com/acl-services/paprika/blob/master/packages/ButtonGroup/src/ButtonGroup.styles.js